package.json

```diff
@@ -83,7 +89,7 @@
         "lint:prettier:fix": "cross-env prettier --write '**/*.md'",
         "lint": "run-p lint:*",
         "typecheck": "tsc --noEmit",
-        "prepublishOnly": "npm run build",
```

Comment:
So that in the release build package.json does not contain engines section

calling the command again `npm pkg delete engines` will not cause errors